src/components/ProductMilestoneCloseResultDetailPage/ProductMilestoneCloseResultDetailPage.tsx

```diff
@@ -141,11 +156,16 @@ export const ProductMilestoneCloseResultDetailPage = () => {
             <PushedBuildsList pushedBuilds={closeResult?.buildPushResults} />
           </ContentBox>
 
-          {/* TODO: Live Log */}
-          <OldUiContentLinkBox
-            contentTitle="Close Result Log"
-            route={`products/${productId}/versions/${productVersionId}/milestones/${productMilestoneId}/close-results/${closeResultId}`}
-          />
+          <Toolbar borderBottom>
+            <ToolbarItem>
+              <TextContent>
+                <Text component={TextVariants.h2}>Logs</Text>
+              </TextContent>
+            </ToolbarItem>
+          </Toolbar>
+          <ContentBox padding>
+            <LogViewer isStatic={closeResult?.status !== 'IN_PROGRESS'} data={logBuffer} />
```

Comment:
is static (not in progress) log fetched line by line?

Reply:
yes, even static logs are passed line by line from the backend. That is the reason I am worrying about the delays. I am trying to preload logs and push them to one update to the logviewer component

Reply:
it should not be much problem after https://github.com/project-ncl/pnc-web-ui-react/pull/578

Reply:
I will keep it like this as we have 8834 and 8838.